pages.ko/common/git-am.md

```diff
@@ -0,0 +1,16 @@
+# git am
+
+> 패치 파일을 적용한다. 이메일로 커밋을 받을 때 유용함. 패치 파일을 생성 할 수 있는 `git format-patch` 또한 참고.
+> 더 많은 정보: <https://git-scm.com/docs/git-am>.
+
+- 패치 파일 적용:
+
+`git am {{path/to/file.patch}}`
```

Comment:
Please translate the placeholders, like `path/to/file.patch`, for get this easier for non-English speakers.

Reply:
Addressed and updated 👍 Thank you @navarroaxel